protocols/grpc/agent.proto

```diff
@@ -116,6 +117,87 @@ message UpdateContainerRequest {
 	LinuxResources resources = 2;
 }
 
+message StatsContainerRequest {
+    string container_id = 1;
+}
+
+message CpuUsage {
+	uint64 total_usage = 1;
+	repeated uint64 percpu_usage = 2;
```

Comment:
Is CPU hotplug supported here? If it is in principle, I think we'll need some tests to ensure we dtrt.

/cc @devimc

Reply:
libcontainer checks the cgroups and each time a new CPU is added, the cgroup is updated, in theory this should work